continuous-integration/travis/install_linux.sh

```diff
@@ -0,0 +1,23 @@
+###############################################################################
+# Install Linux Packages
+###############################################################################
+sudo apt-get upgrade
+sudo apt-get install -qq libatlas-dev liblapack-dev gfortran
+if [[ $TEST_MODE == 'sphinx' ]]; then sudo apt-get install graphviz texlive-latex-extra dvipng; fi
+
+# Install more upto date openjpeg library.
+wget http://openjpeg.googlecode.com/files/openjpeg-1.5.0-Linux-x86_64.tar.gz
+sudo tar -xvf openjpeg-1.5.0-Linux-x86_64.tar.gz --strip-components=1 -C /
```

Comment:
Ok, I see it, it just put it in the top directory... I thought it needed to compile or something...